x-pack/plugins/alerting/server/routes/legacy/create.ts

```diff
@@ -32,15 +32,7 @@ export const bodySchema = schema.object({
   schedule: schema.object({
     interval: schema.string({ validate: validateDurationSchema }),
   }),
-  actions: schema.arrayOf(
-    schema.object({
-      group: schema.string(),
-      id: schema.string(),
-      actionTypeId: schema.maybe(schema.string()),
-      params: schema.recordOf(schema.string(), schema.any(), { defaultValue: {} }),
-    }),
-    { defaultValue: [] }
-  ),
+  actions: actionsSchema,
```

Comment:
they are legacy API, let's not change them, it is like they are deprecated.

Reply:
I'll see if I can figure out how to do this without failing typecheck